crates/wasmedge/src/instance.rs

```diff
@@ -185,89 +152,92 @@ impl Instance for Wasi {
             pidfd: Arc::new(Mutex::new(None)),
         }
     }
-    fn start(&self) -> Result<u32, Error> {
-        let engine = self.engine.clone();
-        let stdin = self.stdin.clone();
-        let stdout = self.stdout.clone();
-        let stderr = self.stderr.clone();
 
+    fn start(&self) -> Result<u32, Error> {
         debug!("preparing module");
-        let spec = load_spec(self.bundle.clone())?;
-
-        let vm = prepare_module(engine, &spec, stdin, stdout, stderr)
-            .map_err(|e| Error::Others(format!("error setting up module: {}", e)))?;
-
-        let cg = oci::get_cgroup(&spec)?;
```

Comment:
Are the oci functions such as `get_cgroup`, `setup_cgroup` etc. used elsewhere, or can we drop them too?

Reply:
Since `libcontainer` supports set-up cgroup, I think don't need to take care of cgroup by runwasi.